rocAL_pybind/amd/rocal/plugin/generic.py

```diff
@@ -29,6 +29,95 @@
 import amd.rocal.types as types
 import ctypes
 
+class ROCALNumpyIterator(object):
```

Comment:
The Numpy iterator seems to be added in both the generic.py and the pytorch.py.

We should think about keeping it here or just as a custom iterator in the respective example pipelines as was done for Coco Readers

@shobana-mcw Please add your comments on this